testsuite/python/ek_interface.py

```diff
@@ -472,13 +539,18 @@ class EKTestWalberlaSinglePrecision(EKTest, ut.TestCase):
 
 @utx.skipIfMissingGPU()
 @utx.skipIfMissingFeatures(["WALBERLA", "CUDA"])
+# TODO: fix bug revealed by heFFTe boundary check
+# (_deps/heffte-src/include/heffte_geometry.h:279)
+@ut.skipIf(EKTest.system.cell_system.get_state()["n_nodes"] != 1,
+           "GPU EK runs for 1 MPI rank")
 class EKTestWalberlaGPU(EKTest, ut.TestCase):
 
     """Test for the Walberla implementation of the EK in double-precision."""
 
     lb_fluid_class = espressomd.lb.LBFluidWalberlaGPU
     ek_lattice_class = espressomd.electrokinetics.LatticeWalberla
     ek_species_class = espressomd.electrokinetics.EKSpeciesGPU
+    ek_solver_class = espressomd.electrokinetics.EKFFTGPU
```

Comment:
@itischler A few issues surfaced when attempting to write extra interface checks for the EKFFTGPU code in `ek_interface.py` and `lattice_vtk.py`. Once this PR gets merged, could you please have a look at it? You will need a Debug build to investigate the error message. I had to run GDB in two terminals with:
```
$ mpiexec -np 2 xterm -fa 'Monospace' -fs 12 -e ./pypresso --gdb \
    ../testsuite/python/ek_interface.py EKTestWalberlaGPU
(gdb) break MPI_Abort
(gdb) run
```